src/components/Swap.tsx

```diff
@@ -66,12 +67,14 @@ const Swap = ({
 
   // Options
   const [optionSlippage, setOptionSlippage] = useState<number>(3)
+  // const [optionInfiniteApproval, setOptionInfiniteApproval] = useState<boolean>(true)
+  const [optionUseLifiContract, setOptionUseLifiContract] = useState<boolean>(true)
 
   // Routes
-  const [routes, setRoutes] = useState<Array<Array<TransferStep>>>([])
+  const [routes, setRoutes] = useState<Array<Array<Step>>>([])
```

Comment:
Shouldn't we maybe use routes here?
```suggestion
  const [routes, setRoutes] = useState<Array<Route>>([])
```

Reply:
This PR has been build before we implemented Routes.